react-accordion-component/src/accordion.jsx

```diff
@@ -0,0 +1,67 @@
+import React from 'react';
+
+export default class App extends React.Component {
+  constructor(props) {
+    super(props);
+    this.state = {
+      onClick: null,
+      isOpen: false
+    };
+
+    this.handleClick = this.handleClick.bind(this);
+  }
+
+  handleClick(e) {
+    const id = e.target.getAttribute('data-id');
+    this.setState({
+      onClick: id,
+      isOpen: true
+    });
+    if (id === this.state.onClick) {
+      this.setState({ isOpen: false });
+    }
```

Comment:
Try modifying your solution to only need one property in state. If the user clicks the same topic again, set the state back to null